### DISCUSSION TITLE
I am a student at Hunan Institute of Technology in Chinese mainland, and I have unsolvable trouble with Github Education Certification

### DISCUSSION THREAD

**Discussion by NyaDoo** [community/community#109465] [GitHub Education]
### Hi everyone,

Hello, I am a software engineering student at Hunan Institute of Technology, Yueyang City, Hunan Province, China.
I successfully completed GitHub's student certification in May 2021, but now with the update of your student certification system, I am having trouble with the secondary certification.

I have a student email address from our school and can provide many supporting materials (including but not limited to student ID card with school official seal, campus card, student transcript provided by the school, student status verification report from the Ministry of Education provided by the Chinese government ).

But even so, due to the limitations of your current system, I still cannot successfully verify my student status.

The main reasons are as follows:

Some of the information is in Chinese, and your system seems to have poor support for Chinese.
The student ID card has multiple pages, but I can only upload and take one photo at a time.
Our government and schools are promoting paperlessness, and some materials are electronic. However, it seems that your system can only upload materials through camera shooting, and cannot directly upload electronic materials.
As we all know, access to Github in mainland China often encounters obstacles. In order to access and upload photos smoothly, I had to use a proxy (VPN), which prevented me from successfully passing address verification.
I am a software engineering student, and the benefits provided by Github student certification are really helpful to me. But now, even if I can provide official materials from the school and the Chinese government’s Ministry of Education, I still cannot successfully complete the student certification.

I think you may be able to make some regional improvements to facilitate our student certification. Apple's education discount certification has cooperated with China's "Alipay". Jetbrains' education certification only requires me to provide a valid certificate issued by the Ministry of Education of China. Student status verification report (electronic version, PDF or photo), and provide the verification code of the student status verification report, which will be manually verified for authenticity and validity on the official website of the Ministry of Education of China.

These are good examples of localization, which allow our Chinese students to successfully complete certification.

I hope I can get a reply，Thanks.

**Comment by namith1003** [comment]
according to the GitHub maintainer in the discussion

https://github.com/orgs/community/discussions/53069

the issue here is not related but is quite in a similar field and I quote their reply below, which you may still be able to follow which would get you better chances of being heard.

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "Other". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.


So use the support link they provided to contact educational support regarding this matter, they will most likely be able to help you regarding this.

Hopefully, this answers your question, feel free to contact me if there is anything else,

A.A Namith Nimlaka

**Comment by NyaDoo** [comment]
I have contacted the education support department and submitted a case, but I have not received a response yet.
This is really frustrating. I understand that GitHub education certification is constantly being abused, so GitHub is constantly tightening its verification measures and reducing the possibility of manual intervention.
However, I believe this should not come at the expense of normal university students.

在 2024年2月23日，12:38，Namith Nimlaka ***@***.***> 写道：

﻿

according to the GitHub maintainer in the discussion

https://github.com/orgs/community/discussions/53069

the issue here is not related but is quite in a similar field and I quote their reply below, which you may still be able to follow which would get you better chances of being heard.

If you believe your applied school's details are incorrect please raise a ticket with Education support <https://support.github.com/contact/education> using the dropdown "Other". 👍 All applications must go through GitHub Global Campus Student Application form<https://education.github.com/discount_requests/student_application>. We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

So use the support link they provided to contact educational support regarding this matter, they will most likely be able to help you regarding this.

Hopefully, this answers your question, feel free to contact me if there is anything else,

A.A Namith Nimlaka

**Reply by namith1003** [reply]
If you have contacted lets wait and see what they respond with there is honestly nothing else that could be done by us at the moment other than hope.
